loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V1.POW

```diff
@@ -31,6 +32,7 @@
 **                         Added fixes for invalid punctuation in disclosure 
 **                         Corrected processing of disclosure blank lines
 **                         Added forced space between consecutive lines of text 
+**    Ver. 1.0.4  08/03/22 R. Robison - Added support for payoff fee display in the UX.
```

Comment:
As this is an enhancement and not a bug fix, wouldn't the next version be 1.1.0?

Reply:
Yes.  Made this change.